README.md

```diff
@@ -21,15 +21,26 @@ $ cd jukebox-js
 
 ## Development
 
-You can run everything in `Docker`. From the root of the repo, you just need to run:
+### Environment
+
+First, create a `.env` file in the root directory. Example vars can be found in the `.env.example` file. At a minimum `WS_MOPIDY_URL`, `WS_MOPIDY_PORT` and `CLIENT_ID` values need to be defined.
+
+`WS_MOPIDY_URL` can point either to a [local](docs/mopidy_install.md) instance of Mopidy or the production/staging instance in the office.
+
```

Comment:
I think I'd prefer that no-one points to the `production` instance TBH. It can have undesirable consequences.

Reply:
Should we make running a local Mopidy mandatory?

Reply:
I think we could setup a staging `mopidy` that anyone could use, but the ideal would be to setup something local.